mongodbatlas/data_source_mongodbatlas_third_party_integration_test.go

```diff
@@ -0,0 +1,292 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"math/rand"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	matlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const (
+	Unknown3rdParty = `
+	resource "mongodbatlas_third_party_integration" "%[1]s" {
+		project_id = "%[2]s"
+		type = "%[3]s"
+	}
+	`
+
+	PAGERDUTY = `
+	resource "mongodbatlas_third_party_integration" "%[1]s" {
+		project_id = "%[2]s"
+		type = "%[3]s"
+		service_key = "%[4]s"
+	}
+	`
+	SLACK = `
```

Comment:
FYI slack no longer works via api

Reply:
Correct but didn't they leave the legacy functionality in the API?   The note about it now needed OAuth is in the "important calls outs" from our docs below.  I guess we could just start w/ this already removed but then we would be different than reality.

Reply:
Hi @gssbzn , @themantissa, I updated the code & docs to deprecate Slack in the plugin.

Reply:
We were not able to "create" a slack integration, I guess the only use of the API is now for editing existing legacy versions, so I don't know how well that may play with users, and how much support that may bring, just flagging as we (mcli) decided to just remove the slack command to avoid people coming after us with "can't create", also the API fails silently, you'll get a 200 but the integration won't be enabled